lighthouse-core/config/default-config.js

```diff
@@ -79,7 +79,8 @@ const UIStrings = {
   seoCategoryManualDescription: 'Run these additional validators on your site to check additional SEO best practices.',
   /* Title of the navigation section within the Search Engine Optimization (SEO) category. Within this section are audits with descriptive titles that highlight opportunities to make a page more usable on mobile devices. */
   seoMobileGroupTitle: 'Mobile Friendly',
-  /* Description of the navigation section within the Search Engine Optimization (SEO) category. Within this section are audits with descriptive titles that highlight opportunities to make a page more usable on mobile devices. */
+  /* Description of the navigation section within the Search Engine Optimization (SEO) category. Within this section are 
```

Comment:
errant hit of the return key? :) this is breaking the strings test (I think maybe we aren't set up to extract multi-line comments into message descriptions) and linting (no trailing spaces)